dev/TreeView/InteractionTests/TreeViewTests.cs

```diff
@@ -853,17 +853,25 @@ private void TreeViewDragAndDropBetweenNodes(bool isContentMode = false)
                 // 5% from the edge of the next item 
                 var distance = (int)(height * 1.45);
 
+                Log.Comment("Click on the item to help make the drag more reliable");
+                dragUIObject.Click();
+
                 Log.Comment("Starting Drag...distance:" + distance);
-                InputHelper.DragDistance(dragUIObject, distance, Direction.South);
+                InputHelper.DragDistance(dragUIObject, distance, Direction.South, 4000);
 
-                ClickButton("GetChildrenOrder");
-                Verify.AreEqual("Root | Root.1 | Root.0 | Root.2", ReadResult());
+                TestEnvironment.VerifyAreEqualWithRetry(
+                    5,
+                    () => "Root | Root.1 | Root.0 | Root.2",
+                    () => {
+                        ClickButton("GetChildrenOrder");
+                        return ReadResult();
+                    });
 
                 ClickButton("GetItemCount");
                 Verify.AreEqual("4", ReadResult());
 
                 ClickButton("GetItemCommonStates");
-                Verify.AreEqual("Selected Selected Selected Selected", ReadResult().Trim());
+                Verify.AreEqual("Normal Normal Normal Normal", ReadResult().Trim());
```

Comment:
Why this changed?

Reply:
Because I changed it to tap on the item instead of the root. And then after the drag&drop operation it's unselected. Maybe that's unexpected?